### PR TITLE
Add travis config

### PR DESCRIPTION
**Pull request by fgarces**
I noticed that this repo didn't have any kind of CI. This PR has the necessary configuration to add travis and test the different versions of activerecord that this gem supports.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  - gemfiles/activerecord_4.gemfile
+  - gemfiles/activerecord_5.gemfile
+  - gemfiles/activerecord_6.gemfile
+services:
+  - postgresql
+before_script:
+  - psql -c 'create database postgresql_cursor_test;' -U postgres
+  - psql -c 'create table products ( id serial primary key, data varchar);' -U postgres -d postgresql_cursor_test
+addons:
+  postgresql: '9.6'
```
Comment (simi) [inline .travis.yml]: Any reason to use 9.6?

**Comment by simi** [comment]
found example build https://travis-ci.org/fgarces/postgresql_cursor, looks ok for me

File: .travis.yml (hunk 1 above)
Comment (fgarces) [reply .travis.yml]: Nope, I copied it from another project. I can update it to the latest version.

**Comment by fgarces** [comment]
@simi Link to the new build using 12.1 https://travis-ci.org/fgarces/postgresql_cursor/builds/638517978

**Comment by simi** [comment]
Looks great!

File: README.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # PostgreSQLCursor for handling large Result Sets
 
 [![Gem Version](https://badge.fury.io/rb/postgresql_cursor.svg)](http://badge.fury.io/rb/postgresql_cursor)
+[![Build Status](https://travis-ci.org/fgarces/postgresql_cursor.svg?branch=master)](https://travis-ci.org/fgarces/postgresql_cursor)
```
Comment (simi) [inline README.md]: This should be changed from `fgarces` to `afair` before merge.

Comment (afair) [reply README.md]: I've just set up Travis for this repo. https://travis-ci.org/afair/postgresql_cursor

**Pull request merged**